Sources/Networking/Core/Upload/MultipartFormDataEncoder.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-open class MultipartFormDataEncoder {
+open class MultipartFormDataEncoder: @unchecked Sendable {
```

Comment:
does this class needs to be open ? There are not even open functions here to override, maybe we should go through more open classes and decide if they need to be kept open 😄 But of course, this is not PR blocking 🙌